### PR TITLE
Add Spock framework and empty test class

### PR DESCRIPTION
**Pull request by KochTobi**

### CODE REVIEW

File: src/test/java/life/qbic/portal/portlet/OMEROClientPortletSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ *
+ * @since: 1.0.0
+ * @author: Tobias Koch
+ */
+class OMEROClientPortletSpec extends Specification {
+
+
+    def "GetPortletContent"() {
+    }
+
```
Comment (jenniferboedker) [inline src/test/java/life/qbic/portal/portlet/OMEROClientPortletSpec.groovy:18]: How do you want to test that?
imo it is sufficient to just create the class, no need to create empty test classes otherwise they might stay empty

File: pom.xml [hunk 2]
```diff
@@ -87,8 +87,14 @@
 			<artifactId>commons-codec</artifactId>
 			<version>1.4</version>
 		</dependency>
+    <dependency>
+      <groupId>org.spockframework</groupId>
+      <artifactId>spock-core</artifactId>
+      <version>1.2-groovy-2.5</version>
```
Comment (jenniferboedker) [inline pom.xml:93]: is it 1.2 or 1.3-groovy-2.5? I have 1.3 for MetaStats.

**Review by jenniferboedker — APPROVED** [review]
some minor comments but lgtm :)

File: src/test/java/life/qbic/portal/portlet/OMEROClientPortletSpec.groovy (hunk 1 above)
Comment (KochTobi) [reply src/test/java/life/qbic/portal/portlet/OMEROClientPortletSpec.groovy:18]: You are right. I let my IDE create the tests.
```suggestion
```
You can go ahead and commit this suggestion so we both get credit.

File: pom.xml (hunk 2 above)
Comment (KochTobi) [reply pom.xml:93]: Good question. I did let IntelliJ add it to the pom for me.
```suggestion
      <version>1.3-groovy-2.5</version>
```

File: src/test/java/life/qbic/portal/portlet/OMEROClientPortletSpec.groovy (hunk 1 above)
Comment (jenniferboedker) [reply src/test/java/life/qbic/portal/portlet/OMEROClientPortletSpec.groovy]: Thanks!
